test/routes/graphql/documentNodes.ts

```diff
@@ -647,9 +647,86 @@ export const Mutation_createPresignedUrl = gql(`
     }
   }
 `);
+export const Mutation_createChat = gql(`
+  mutation CreateChat($input: MutationCreateChatInput!) {
```

Comment:
```suggestion
  mutation Mutation_createChat($input: MutationCreateChatInput!) {
```

Please follow the existing naming convention